webpack.config.babel.js

```diff
@@ -107,9 +105,13 @@ const PROD_CONFIG = {
   'MOBILE_URL' : 'https://id.onfido.com',
   'SMS_DELIVERY_URL': 'https://telephony.onfido.com',
   'PUBLIC_PATH' : `https://assets.onfido.com/web-sdk-releases/${packageJson.version}/`,
+  WOOPRA_DOMAIN
 }
 
-const TEST_CONFIG = { ...PROD_CONFIG, PUBLIC_PATH: '/', 'MOBILE_URL' : '/' }
+const TEST_CONFIG = { ...PROD_CONFIG,
```

Comment:
Not introduced in this PR, but It worries me that we are using the same config for TEST and PROD environments.

Reply:
the same endpoints, yes, but not the same config per se.
I agree in principle, but practically staging causes too much flakiness